inspirehep/records/marshmallow/literature/common/author.py

```diff
@@ -13,7 +13,7 @@
 
 class AuthorSchemaV1(Schema):
     affiliations = fields.Raw()
-    alternative_names = fields.Raw()
```

Comment:
actually if you check hep schema `references.reference` part, this is `alternative_names`.

Reply:
But this is below `name`, not `references.reference`